sql/src/test/java/io/crate/integrationtests/UpdateIntegrationTest.java

```diff
@@ -985,11 +985,20 @@ public void test_update_by_query_returning_multiple_results() throws Exception {
         assertEquals(1, response.rowCount());
         refresh();
 
-        execute("update test set message='updated' where message='msg' and x > 0 returning message as message_renamed");
+        execute("update test set message='updated' where message='msg' and x > 0 " +
+                "returning _docid, _seq_no as seq, message as message_renamed");
 
-        assertThat((response.rowCount()), is(2L ));
-        assertThat((response.cols()[0]), is("message_renamed"));
-        assertThat(response.rows()[0][0], is("updated"));
-        assertThat(response.rows()[1][0], is("updated"));
+        assertThat((response.rowCount()), is(2L));
+        assertThat((response.cols()[0]), is("_docid"));
+        assertThat((response.cols()[1]), is("seq"));
+        assertThat((response.cols()[2]), is("message_renamed"));
+
+        assertThat(response.rows()[0][0], is(0));
+        assertThat(response.rows()[0][1], is(0L));
```

Comment:
Shouldn't this be > 0, as _seq_no starts out at 0 initially?

Reply:
Good point and nice catch. The return values are actually generated before the seq_no is incremented which is wrong. This is needs to be fixed. Thank you!